src/Orleans/Streams/SimpleMessageStream/SimpleMessageStreamProducerExtension.cs

```diff
@@ -23,16 +25,16 @@ internal class SimpleMessageStreamProducerExtension : IStreamProducerExtension
         private readonly IStreamPubSub              streamPubSub;
         private readonly bool                       fireAndForgetDelivery;
         private readonly bool                       optimizeForImmutableData;
-        private readonly Logger                     logger;
+        private readonly ILogger                     logger;
 
-        internal SimpleMessageStreamProducerExtension(IStreamProviderRuntime providerRt, IStreamPubSub pubsub, bool fireAndForget, bool optimizeForImmutable)
+        internal SimpleMessageStreamProducerExtension(IStreamProviderRuntime providerRt, IStreamPubSub pubsub, ILoggerFactory loggerFac, bool fireAndForget, bool optimizeForImmutable)
         {
```

Comment:
There's several occurrences of loggerFac sill

Reply:
will fix it. I thought that was the only occurrence